shared/specs/model/exercise/question.spec.js

```diff
@@ -78,9 +76,9 @@ describe('Exercise Question', () => {
         question.collaborator_solution_html = '';
         expect(question.collaborator_solutions).toHaveLength(0);
     });
-    it('uses answers for detecting multi-choice', () => {
+    it('uses formats for detecting multi-choice', () => {
         expect(question.isMultipleChoice).toBe(true);
-        question.answers = [];
+        question.formats = ['free-response'];
         expect(question.isMultipleChoice).toBe(false);
```

Comment:
we do not want to switch to using formats for determining multiple-choice.  that should continue to rely on if the exercise has any answers, that way it matches what tutor-server does.  cc @Dantemss to confirm the BE still does that though.

This task was about clearing the answers array when the exercise is saved with it's format not set to multiple-choice